dev.env

```diff
@@ -22,7 +22,7 @@
 
 source ./build.env
 
-export VTPORTSTART=15000
+export VTPORTSTART=6700
```

Comment:
I'm wondering if this will break the local example, but it seems to be passing 🤷 
@dbussink can you verify that you can `source dev.env` and then go to examples/local and run the example without problems?

Reply:
This is the same value as what we use if this env var is not set.